docs/unix/quickref.rst

```diff
@@ -0,0 +1,93 @@
+.. _unix_quickref:
+
+Quick reference for the UNIX and Windows ports
+==============================================
+
+Command line options
+--------------------
+
+Usage::
+
+    micropython [ -h ] [ -i ] [ -O<level> ] [ -v ] [ -X <option> ] [ -c <command> | -m <module> | <script> ] [ <args> ]
+
+
+Invocation options:
+
+.. option:: -c <command>
+
+   Runs the code in ``<command>``. The code can be one or more Python statements.
+
+.. option:: -m <module>
+
+   Runs the module ``<module>``. The module must be in ``sys.path``.
+
+.. option:: <script>
+
+   Runs the file ``<script>``. The file can be a plain text ``.py`` file or
+   a compiled ``.mpy`` file.
```

Comment:
I'm pretty sure you can't run .mpy files this way... you can only do it using `-m`

Reply:
Indeed. Fixed.